frontend/src/Modules/Certificate/Components/CertificateList/CertificateList.js

```diff
@@ -65,27 +69,35 @@ class CertificateList extends Component {
         })
 
     }
+    answer = (isYes) => {
+        const { id } = this.state
+        if (isYes) {
+            console.log(id)
```

Comment:
what is this?

Reply:
cái components này lúc trước có dùng nhưnng giờ ko dùng nữa a ạ .nhưng code vẫn để lại .Đoạn này e ko rõ vì a hiền viết nhưng ko sao đâu anh cái này ko ảnh hưởng tới form khác